src/plugins/intel_gpu/src/runtime/layout.cpp

```diff
@@ -112,13 +115,37 @@ std::vector<tensor::value_type> layout::get_dims() const {
 }
 
 std::vector<tensor::value_type> layout::get_padded_dims() const {
-    if (is_dynamic())
-        throw std::runtime_error("[GPU] get_padded_dims() is called for dynamic shape");
+    OPENVINO_ASSERT(!is_dynamic() || has_upper_bound(), "[GPU] get_tensor() is called for dynamic shape without upper bound");
+    ov::Shape shape;
+    if (is_dynamic() && has_upper_bound()) {
+        for (const auto& dim : size) {
+            shape.push_back(dim.get_max_length());
```

Comment:
`shape = size.get_max_shape()` ?

Reply:
I simply copy code from get_tensor(). What's the problem here? Do you mean get_max_length?

Reply:
No, I mean replacing loop with single line, as PartialShape::get_max_shape() does same thing as I can see

Reply:
done